strikt-gradle/src/main/kotlin/strikt/gradle/testkit/BuildResult.kt

```diff
@@ -0,0 +1,41 @@
+package strikt.gradle.testkit
+
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.BuildTask
+import org.gradle.testkit.runner.TaskOutcome
+import strikt.api.Assertion
+
+/**
+ * Maps this assertion to the task at the provided [taskPath].
+ * @see BuildResult.task
+ */
+fun <T : BuildResult> Assertion.Builder<T>.task(taskPath: String): Assertion.Builder<BuildTask?> =
+  get("task path") { task(taskPath) }
+
+/**
+ * Maps this assertion to the task paths of the build with the provided [outcome].
+ * @see BuildResult.taskPaths
+ */
+fun <T : BuildResult> Assertion.Builder<T>.taskPaths(outcome: TaskOutcome): Assertion.Builder<List<String>> =
+  get("task paths with outcome $outcome") { taskPaths(outcome) }
+
+/**
+ * Maps this assertion to an assertion on all of the tasks from the build.
+ * @see BuildResult.getTasks
+ */
+fun <T : BuildResult> Assertion.Builder<T>.tasks(): Assertion.Builder<List<BuildTask>> = get("all tasks", BuildResult::getTasks)
```

Comment:
Would this be better as a `val` rather than a `fun`? Then it would mirror the Gradle API (like `output` below).